sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,995 @@
+# Preamble
+
+SIP Number: 022
+
+Title: sBTC: Design of a Decentralized Two-way Peg for Bitcoin
+
+Authors:
+* Aaron Blankstein <aaron@hiro.so>
+* Alie Slade
+* Andre Serrano <andre@resident.stacks.org>
+* Ashton Stephens <ashton@trustmachines.co>
+* Brice Dobry <brice@hiro.so>
+* Don Park <donp@trustmachines.co>
+* Fernando Foy <fernando@trustmachines.co>
+* Friedger Müffke <mail@friedger.de>
+* Igor Sylvester <igor@trustmachines.co>
+* Jacinta Ferrant <jacinta@trustmachines.co>
+* Jesus Najera <jnajera1917@gmail.com>
+* Joey Yandle <joey@trustmachines.co>
+* Jude Nelson <jude@stacks.org>
+* Marvin Janssen <marvin@ryder.id>
+* Mike Cohen <mjoecohen@gmail.com>
+* Sayak Chatterjee <sayak@trustmachines.co>
+* Stjepan Golemac <stjepan@trustmachines.co>
+* Tycho Onnasch <tycho@tychoonnasch.com>
+
+Consideration: Technical, Governance
+
+Type: Consensus
+
+Status: Draft
+
+Created: 28 September 2023
+
+License: CC0-1.0
+
+Sign-off:
+
+Layer: Consensus (hard fork)
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+This SIP proposes the creation of a **trust-minimized 2-way peg** in which BTC (the
+tokens of the current Stacks burnchain) can be manifested on Stacks as a
+SIP-010-compliant fungible token, called sBTC, and later redeemed for the same
+number of BTC.  This proposal redefines the act of Stacking to be the act of
+maintaining an _sBTC wallet_, which is used to hold BTC and process sBTC
+redemptions.  Stackers are collectively responsible for ensuring that all
+liquid sBTC are backed by an equal number of BTC, and that redemptions are
+processed in a timely manner.  As a reward, they continue to receive PoX
+payouts.
+
+# Introduction
+
+## Glossary
+
+| Term | Definition |
```

Comment:
Add `Reward cycle` with a link to SIP-007 as requested in https://github.com/stacksgov/sips/pull/156#discussion_r1361225487